general/development/process/release/index.md

```diff
@@ -23,9 +23,8 @@ sidebar_position: 7
 | :---: | :---: | :---: | --- | --- |
 | 1. | &#10003; |  | The [Continuous Integration](../integration/index.md#during-continuous-integration.2ffreeze.2fqa-period) period begins. Warn about it everywhere (telegram, exposed posts...).<br/>Rolling (on demand, beta, rc... all them together with stable weeklies) happens often (Tuesday & Friday are the usual days). All the team is on integration 100% since this week and until the end of continuous. | Integration Team |
 | 2. | &#10003; |  | Full demo of new code and sign-off for internal HQ projects. Decide which projects will be completed by the code freeze. | Platform manager / Head of educational solutions |
-| 3. | &#10003; |  | Check [closed qa_test_required-labelled issues](https://tracker.moodle.org/issues/?jql=labels%20%3D%20qa_test_required%20AND%20status%20%3D%20Closed) and create new QA tests as required. | Testing Maintainer |
```

Comment:
Doe this part of our process no longer exist then?

Reply:
Ah - I see... are these merged now? Do they need to be the same step? Isn't it just a case of changing from "Community manager" to "Testing Maintainer"?

Reply:
I think having both in the same sections makes it easier to read